readme.md

```diff
@@ -44,6 +46,23 @@ end function
 
 For a working sample app see the "test" folder. Its contents can be zipped for
  installation as a dev channel on a Roku.
+ 
+# Additional information
+
+## A differences between `roUrlTransfer` and `roStreamSocket`
+
+The `roUrltransfer` is sending bigger data faster, Roku replied to this:
+> Could be the difference between using curlLibrary and lower-level roStreamSocket implementation. The curl library is highly optimized as we use it for all the manifest handling in streaming.
+
+The `roStreamSocket` is separated into standalone thread but still is serialized, a devices handling higher amount of data could overloaded a buffer for some moments and data could be send longer. 
```

Comment:
> a devices handling higher amount of data could overloaded a buffer for some moments and data could be send longer. 

devices handling a higher amount of data could overload a buffer for some moments and data could send for longer.